Modules/IndividualAssessment/classes/class.ilIndividualAssessmentMembersTableGUI.php

```diff
@@ -259,24 +265,47 @@ protected function getGradedInformations(ilDateTime $event_time): array
 	 *
 	 * @return string[]
 	 */
-	protected function getExaminerLogin(int $examiner_id = null): array
+	protected function getFullNameFor(int $user_id = null): string
 	{
-		if(is_null($examiner_id)) {
+		if(is_null($user_id)) {
+			return "";
+		}
+
+		$name_fields = ilObjUser::_lookupName($user_id);
+		$name = $name_fields["lastname"].", ".$name_fields["firstname"]." [".$name_fields["login"]."]";
+
+		return $name;
+	}
+
+	protected function getPortfolioLink(string $case, string $examiner, int $examiner_id)
```

Comment:
This is a misnomer. We link to the "Profile" not the "Portfolio". These are two different things.